auth/service.hh

```diff
@@ -174,6 +179,10 @@ public:
         return _qp;
     }
 
+    future<> announce_mutations(::service::mutations_collector& mc) {
+        return mc.announce(_group0_client, "auth", _as, ::service::raft_timeout{});
```

Comment:
same as with last PR, I am not really sure what's the use of this label, is it worth to trouble with passing around the description string?

I didn't do it because in the future it prevents unification. Currently announce_mutations is called in each statement but the follow-up can be that the whole guard is replaced with mutation_collector and then announced in a single place, in such case not possible to say which statement it is.

Reply:
> same as with last PR, I am not really sure what's the use of this label, is it worth to trouble with passing around the description string?
> 

It is added to group0 history table which allows to see the entire leniarized history of all group0 operations.

Reply:
that much I know, but how it's useful?

Reply:
Don't you think it would be useful for debug?

Reply:
I don't think burden of proof is on my side here, to prove something is usefully first prove it, not ask to prove that it is not usefull :) 

but seriously my reasoning is that we should also have some way to view mutations, no? Then if you know that it's auth and see mutations that's all you need.

Reply:
fixed